www/templates/logs.html

```diff
@@ -19,12 +19,24 @@
         <div class="container">
             <ul class="list-unstyled">
                 <li>
-                    <a href="{{ iframe_url }}" target="_blank">Open in new window</a>
+                  <a href="{{ iframe_url }}" target="_blank">Open in new window</a>
                 </li>
                 <li>
                     /
                 </li>
                 <li>
+		  <a href="{{ plaintext_url }}" target="_blank">Download as plaintext </a>
```

Comment:
Remove `target="_blank"` otherwise each time you click link browser will try to open new tab.